packages/react/src/components/payStream/time-units.ts

```diff
@@ -0,0 +1,11 @@
+import { IntervalType } from '@heliofi/common';
+
+export const timeUnitLabels = {
```

Comment:
maybe putting this in '@heliofi/common'; would be better

Reply:
maybe SDK? it's not used in backend.